resolwe_bio/processes/WGBS/methcounts.yml

```diff
@@ -11,7 +11,7 @@
       docker:
         image: resolwebio/wgbs:1.0.0
```

Comment:
Please bump the process to the latest version of the `resolwebio/wgbs` to minimise the usage of different versions of Docker images.